tests/test_release.py

```diff
@@ -38,6 +38,9 @@ def test_release_schedule_adheres_to_schema(
 
     for date, release_version in release_schedule.items():
 
+        # No old releases should exist within the release schedule (deleted during release process)
+        assert date >= today
+
```

Comment:
The extra security is a nice-to-have. The `ge_releaser` tool has some input validation but bolstering this test can't hurt.

Reply:
❤️